platform/platform-shared/pom.xml

```diff
@@ -21,6 +21,18 @@
       <version>1.8.8-R0.1-SNAPSHOT</version>
       <scope>provided</scope>
     </dependency>
+<!--    <dependency>-->
```

Comment:
Is this supposed to be a reminder for something?

Reply:
This was something i used during development to have the IDE find me 1.20.6 classes to check what was diff
Can remove